### DISCUSSION TITLE
How to Install Open-Shell-Menu

### DISCUSSION THREAD

**Discussion by BillyJC** [Open-Shell/Open-Shell-Menu#953] [Q&A] [answered]
Hello:

I downloaded "Open-Shell-Menu-master" to restore classic start menu in windows 11, but I can' figure out how to install it.
Please let me know how to do it?

Billy

**Comment by bonzibudd** [comment, marked as answer]
Download the installer from the Releases page: https://github.com/Open-Shell/Open-Shell-Menu/releases

**Reply by BillyJC** [reply]
Thanks. I did download it and it works great!
